src/SpineRuntime/Animation.js

```diff
@@ -0,0 +1,70 @@
+var spine = require('../SpineRuntime') || {};
+spine.Animation = function (name, timelines, duration)
```

Comment:
This bit isn't necessary, just export the class itself, and directly require the needed classes. This isn't really following the CJS pattern.

For example:

```js
function Animation(name, timelines, duration) {
    // ...
}

module.exports = Animation;

Animation.prototype = {
    // ...
};
```

The index.js file already aggregates these files, no need to do this weird double object export.